libcxx/include/complex

```diff
@@ -105,32 +103,14 @@ public:
 };
 
 template<>
-class complex<long double>
 {
 public:
-    typedef long double value_type;
 
-    constexpr complex(long double re = 0.0L, long double im = 0.0L);
     constexpr complex(const complex<float>&);
     constexpr complex(const complex<double>&);
 
-    constexpr long double real() const;
-    void real(long double);
-    constexpr long double imag() const;
-    void imag(long double);
-
-    complex<long double>& operator=(const complex<long double>&);
-    complex<long double>& operator= (long double);
-    complex<long double>& operator+=(long double);
-    complex<long double>& operator-=(long double);
-    complex<long double>& operator*=(long double);
-    complex<long double>& operator/=(long double);
-
-    template<class X> complex<long double>& operator= (const complex<X>&);
-    template<class X> complex<long double>& operator+=(const complex<X>&);
-    template<class X> complex<long double>& operator-=(const complex<X>&);
-    template<class X> complex<long double>& operator*=(const complex<X>&);
-    template<class X> complex<long double>& operator/=(const complex<X>&);
+
+
 };
```

Comment:
This diff looks weird; shouldn't the whole block just be gone? (I get that it's in the documentation comment at the top, it';s still weird.)

Reply:
Actually remembering my overarching comment now, maybe also wrap it with ifdefs inside this comment ;)

Reply:
Caught me there, thanks. I just did a line by line removal and forgot about the specialization.